lib/sle2docker/native_image.rb

```diff
@@ -34,35 +34,36 @@ def activate
     private
 
     def compute_metadata_file
+      match = parse_image_filename(@image_name)
+      @metadata = parse_metadata_file("#{match['metadata_file']}.metadata")
+
+      @repository   = @metadata['image']['name']
+      @tag          = @metadata['image']['tags'][0]
+      @build        = match['build']
+      @image_id     = "#{@repository}:#{@tag}"
+      @options['tag_with_build'] && @image_id << '-' + @build
```

Comment:
This looks rather hackish 😅 For `@image_id << '-' + @build`, prefer:

```ruby
"#{@image_id}-#{@build}"
```

Reply:
Agree, I just followed the RootFSImage syntax style here, but I am completely fine with that change.